### PR TITLE
Fallback to JSON

### PR DESCRIPTION
**Pull request by andrewhood125**

### CODE REVIEW

**Comment by svpernova09** [comment]
:+1: I think this is a decent compromise of #229

File: Vagrantfile [hunk 1]
```diff
@@ -15,7 +16,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 		config.vm.provision "file", source: aliasesPath, destination: "~/.bash_aliases"
 	end
 
-	Homestead.configure(config, YAML::load(File.read(homesteadYamlPath)))
+    if File.exists? homesteadYamlPath then
```
Comment (taylorotwell) [inline Vagrantfile]: Indentation is off here.

Comment (andrewhood125) [inline Vagrantfile]: One sec.

**Comment by andrewhood125** [comment]
@taylorotwell Updated to use tabs.

**Comment by leonhh** [comment]
It is indeed a decent solution :+1:

**Pull request merged**